ci-operator/step-registry/baremetal/lab/pre/serial-console/baremetal-lab-pre-serial-console-commands.sh

```
@@ -23,9 +23,9 @@ for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
 
     case "${model}" in
       "PowerEdge R740")
-        if [[ $(echo "$bios_attributes" | jq -r '.SerialPortAddress') != "Serial1Com2Serial2Com1" ]] || [[ $(echo "$bios_attributes" | jq -r '.ExtSerialConnector') != "Serial2" ]]; then
+        if [[ $(echo "$bios_attributes" | jq -r '.SerialComm') != "OnConRedirAuto" ]] || $(echo "$bios_attributes" | jq -r '.SerialPortAddress') != "Serial1Com2Serial2Com1" ]] || [[ $(echo "$bios_attributes" | jq -r '.ExtSerialConnector') != "Serial2" ]]; then
```

Shellcheck (0.11.0)

[warning] Found trailing ]] outside test. Add missing [[ or quote if intentional.

(SC2171)